<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generalMainPage</class>
 <widget class="GeneralMainPage" name="generalMainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelTypeNumber">
         <property name="text">
          <string>EMPTY NUMBER!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editNumber"/>
       </item>
       <item>
        <widget class="QLabel" name="labelNumberError">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelTypeDate">
         <property name="text">
          <string>EMPTY DATE!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editDate"/>
       </item>
       <item>
        <widget class="QLabel" name="labelDateError">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Kundennummer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editCustomerNumber"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Adressen (F1)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Anrede:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editSalutation"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Stra√üe:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editStreet"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>PLZ / Ort:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editPlace"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Betreff:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editSubject"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="specialDataLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Skonto (%):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editSkonto"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Zahlung Tg. norm./Sko.:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editPayNormal">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editPaySkonto">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Stundensatz:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editHourlyRate">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Kopftext</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PageTextEdit" name="editHeading">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Standardtext (F5)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Schlusstext</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PageTextEdit" name="editEnding">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Standardtext (F6)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeneralMainPage</class>
   <extends>QWidget</extends>
   <header>pages/general_main_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageTextEdit</class>
   <extends>QTextEdit</extends>
   <header>functionality/utils.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editNumber</tabstop>
  <tabstop>editDate</tabstop>
  <tabstop>editCustomerNumber</tabstop>
  <tabstop>editSalutation</tabstop>
  <tabstop>editName</tabstop>
  <tabstop>editStreet</tabstop>
  <tabstop>editPlace</tabstop>
  <tabstop>editSubject</tabstop>
  <tabstop>editSkonto</tabstop>
  <tabstop>editPayNormal</tabstop>
  <tabstop>editPaySkonto</tabstop>
  <tabstop>editHourlyRate</tabstop>
  <tabstop>editHeading</tabstop>
  <tabstop>editEnding</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>generalMainPage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>generalMainPage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CopyData(QString)</slot>
  <slot>TakeFromMaterial()</slot>
  <slot>TakeFromService()</slot>
  <slot>MakeNewEntry()</slot>
  <slot>TakeFromAdress()</slot>
  <slot>TakeDefaultHeading()</slot>
 </slots>
</ui>
